lib/Listeners/CalendarObjectCreatedListener.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/**
+ * @copyright Copyright (c) 2021 Paweł Kuffel <pawel@kuffel.io>
+ *
+ * @author Paweł Kuffel <pawel@kuffel.io>
+ *
+ * @license GNU AGPL version 3 or any later version
```

Comment:
Feel free to modify the copyright notice so as to list yourself as the author in this file.